includes/annotationcommand_adapter.inc

```diff
@@ -470,7 +470,17 @@ function annotationcommand_adapter_add_hook_options(&$command)
   // the extension hook a core command, or have an early-loaded global
   // extension hook a late-loaded extension (e.g. attached to a module).
   $names = annotationcommand_adapter_command_names($command);
-  $names[] = '*'; // we are missing annotations here; maybe we just don't support that? (TODO later, maybe)
+  $names[] = '*';
+  // Add possible hooks.
+  $blacklist = ['command', 'handle-remote-commands', 'strict-option-handling', 'allow-additional-options', 'bootstrap', 'topic', 'hidden', 'hidden-option'];
```

Comment:
Props for removing annotations we know will not be used as hooks. Maybe should add that to the upstream api too.